### ISSUE TITLE
Feature request: command line option to opt for string literals for OOV words

### ISSUE THREAD

**Issue by inariksit** [GrammaticalFramework/gf-ud#24]
(previously the other half of #22 , split into its own issue)

### String literals for OOV words
If the sentence contains words that are not in the lexicon, I would like to create those words as string literals. So "mimsy were the borogroves" would result in an otherwise normal GF tree, but with the subtrees `StrA "mimsy"` and `StrN "borogrove"`.

This feature should be optional: either command line arg, or check if the grammar contains `StrA : String -> A`.

### Future work: modify the PGF grammar?
The new majestic runtime will allow modifying PGFs on the fly. So when that is possible, I'd prefer to create proper lexicon entries `mimsy_A` and `borogrove_N`, using real GF smart paradigms, and insert them into the PGF. 
(Similarly, the Backup* funs from #22 would also be possible to insert into the PGF.)

So once the new runtime is in place, I think that command line argument would be a better option. And if these features are added into gf-ud already before majestic runtime, it makes sense to just use command line arguments from the beginning.
